sync-todo/v2/client/maui/RealmTodo/Services/RealmService.cs

```diff
@@ -1,22 +1,41 @@
-﻿using Realms;
+﻿using System.Text.Json;
+using Realms;
 using Realms.Sync;
 using RealmTodo.Models;
 
 namespace RealmTodo.Services
 {
     public static class RealmService
     {
-        private const string appId = "****";
+        private static bool serviceInitialised;
 
         private static Realms.Sync.App app;
 
         private static Realm mainThreadRealm;
 
         public static User CurrentUser => app.CurrentUser;
 
-        public static void Init()
+        public static async Task Init()
         {
-            app = Realms.Sync.App.Create(appId);
+            if (serviceInitialised)
```

Comment:
To keep the class tidy, you might be able to declare serviceInitialised as a local static instead of a class member?

Reply:
I am not sure of what you mean here.

Reply:
No, I mean local variable. You can scope it to the method itself, by declaring it within the method as static.

Reply:
Ah! If I understood you correctly, I think that cannot be done in C#, static variables cannot be declared inside methods like in C for example

Reply:
I see! Thanks.